wd_helper.au3

```diff
@@ -728,6 +754,7 @@ Func _WD_LoadWait($sSession, $iDelay = Default, $iTimeout = Default, $sElement =
 
 		If $sElement <> '' Then
 			_WD_ElementAction($sSession, $sElement, 'name')
+			$iErr = @error
```

Comment:
shole we here need to avoid multile errors in the same way like it was done with `_WD_FindElement` usage in `_WD_WaitElement` ?

Reply:
No, and I believe that $iErr was intentionally not saved here to avoid exiting the While loop. By adding this line, you will force the loop to prematurely exit when $sElement parameter is being processed.

If you agree, then I would recommend replacing your latest addition with a comment explaining why we don't update $iErr after the call to _WD_ElementAction

Reply:
Take a look on:
[_WD_LoadWait prevent multiple errors](https://github.com/Danp2/au3WebDriver/pull/290/commits/2c552e500c04d1c4d8774a8741e55f94666fb9ea)

I test this mod with this modified example:
```autoit
Func DemoSleep()
	; enable Abort button
	GUICtrlSetState($__g_idButton_Abort, $GUI_ENABLE)

	; set up outer/user specific sleep function to take control
	_WD_Option("Sleep", _USER_WD_Sleep)

	_WD_Navigate($sSession, "https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Win/")
	Local $iError = @error
	If Not $iError Then
		Local $sElement = _WD_WaitElement($sSession, $_WD_LOCATOR_ByXPath, '//img[@class="loader-spinner" and @ng-show="loading"]', Default, 3 * 1000)
		If Not @error Then
			; it can take a long time to load full content of this webpage
			; this following function is waiting to the progress spinner will not exist
			_WD_LoadWait($sSession, Default, 3 * 60 * 1000, $sElement)
		EndIf
		$iError = @error

		; normaly it will wait as webpage will load full content (hidden spinner) or will end with TimeOut
		; but thanks to using _WD_Option("Sleep", _USER_WD_Sleep) you can abort waiting by clicking scecial Abourt button or by clicking X closing button on the "Webdriver Demo" GUI window
	EndIf

	; disable Abort button
	GUICtrlSetState($__g_idButton_Abort, $GUI_DISABLE)

	; set up internal sleep function - back to standard route
	_WD_Option("Sleep", Sleep)

	Return SetError($iError)
EndFunc   ;==>DemoSleep
```